### PR TITLE
Input

### PR DESCRIPTION
**Pull request by Ananyamadhu08**
- styled input classes.
- created different color variations for input.
- created different hover classes.
- added animation and created focus classes. 
Screen Shot 2022-02-10 at 12 37 37 PM
Screen Shot 2022-02-10 at 12 46 06 PM
Screen Shot 2022-02-10 at 12 46 34 PM

### CODE REVIEW

File: css/input.css [hunk 1]
```diff
@@ -0,0 +1,496 @@
...
+.input {
+  box-sizing: border-box;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  border: 2px solid var(--grey-500);
+  border-radius: 0.5rem;
+  /* color: white; */
```
Comment (VedanthB) [inline css/input.css:15]: delete commented code

**Review by VedanthB — APPROVED** [review]
looks good to go 🚀

File: css/input.css [hunk 2]
```diff
@@ -0,0 +1,496 @@
...
+  border-color: var(--stone-500);
+}
+
+.input-grey-focus:focus {
+  border-color: var(--grey-500);
+}
+
+.input-slate-focus:focus {
+  border-color: var(--slate-500);
+}
```
Comment (ashishkatiyar2404) [inline css/input.css:496]: input.css 💯 you are consistent with variable nice

**Review by ashishkatiyar2404 — APPROVED** [review]

**Comment by ashishkatiyar2404** [comment]
looks good to merge.

**Pull request merged**